### PR TITLE
Init project and add landing page with dummy data

### PR DESCRIPTION
**Pull request by kochch1337**
Landing page fetches data from dummy data api and shows solution names in a list

### CODE REVIEW

File: helpers/api.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    SupportGroup: "supp-einkauf",
+    Modules: [
+      {
+        Modul: "ABC",
+        Developers: [
+          {
+            DevloperId: "1",
+            Firstname: "Thomas",
+            Lastanme: "Müller",
+          },
```
Comment (marcbendixen) [inline helpers/api.js:20]: I recommend you to write the keys in `camelCase`. It's a naming convention that is used by most APIs etc. Also for later use in your code it would make things look better and it's less confusing (example: `<li key={data.id}>{data.solution}</li>` instead of `<li key={data.Id}>{data.Solution}</li>`

```suggestion
    id: "1",
    solution: "Einkauf",
    team: "AA",
    bpe: "Sebastian Brucker",
    bseint: "Jochen Maier",
    bsegr: "Helmut Schön",
    leadDevloper: "Veit Maurer",
    cbo: "Ahmet Peros",
    supportGroup: "supp-einkauf",
    modules: [
      {
        modul: "ABC",
        developers: [
          {
            devloperId: "1",
            firstName: "Thomas",
            lastName: "Müller",
          },
```

**Review by marcbendixen — CHANGES_REQUESTED** [review]
Good job 🚀 
I just made a comment regarding the naming convention of keys.

File: helpers/api.js (hunk 1 above)
Comment (kochch1337) [reply helpers/api.js:20]: Thank you. I will updagte this in the next feature branch

Comment (marcbendixen) [reply helpers/api.js:20]: Normally you should update it in this PR if you follow the Git Workflow, so don't forget about it 😉

File: helpers/api.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  return data.find((solution) => solution.id === id);
+}
+
+const hostname = "http://localhost:3000/";
+
+function fetcher(url) {
+  if (!url) return;
+
+  return fetch(hostname + url).then((res) => res.json());
+}
```
Comment (shebtastic) [inline helpers/api.js:305]: this code doesn't seem to be needed here. as such it should not be part of this PR.

File: pages/index.js [hunk 3]
```diff
@@ -1,3 +1,25 @@
-export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+import { fakeFetcher } from "../helpers/api";
+import useSWR from "swr";
+const apiUrl = "/api/data";
```
Comment (shebtastic) [inline pages/index.js:3]: why is this static string a variable, instead of inlined?

using the static string directly as an inline argument is more idiomatic.

File: helpers/api.js [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+function fakeFetcher(url) {
+  if (!url) return;
+
+  switch (true) {
+    case "/api/data" === url:
+      return getAllData();
+    case url?.startsWith("/api/data/"):
+      return getDataById(url.split("/").at(-1));
```
Comment (shebtastic) [inline helpers/api.js:314]: this isn't in use either

File: pages/index.js [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+
+  if (error) {
+    return <h1>Data could not be fetched</h1>;
+  }
+
+  if (!data) {
+    return <h1>loading</h1>;
+  }
+  return (
+    <>
```
Comment (shebtastic) [inline pages/index.js:16]: the pages are missing the `main` tag. this is not required but highly recommended. you should either switch out the fragment with a `main` tag here or wrap it in `_app.js`

File: helpers/api.js (hunk 1 above)
Comment (shebtastic) [reply helpers/api.js:20]: i agree with marc. maybe create a fix branch in the meantime and post a new PR for that branch, before continuing on the feature branch. that way we can minimize divergence.

File: pages/index.js [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+      <h1>Solutions</h1>
+      <ul>
+        {data?.map((solution) => {
```
Comment (shebtastic) [inline pages/index.js:19]: `data.map` doesn't need to be optionally chained, if there is an early return on `!data`
